pipelines/scripts/clean_notebooks.py

```diff
@@ -0,0 +1,49 @@
+from pipelines.scripts.util import Util
+import json
+from typing import List, Dict, Any
+import logging
+
+
+"""
+    Clean all notebooks in the local synapse workspace
+    Run using `python3 pipelines/scripts/clean_notebooks.py`
+    This can be used to cleanup any issues raised by `pipelines/scripts/check_notebooks_valid.py`
+"""
+
+logging.basicConfig(level=logging.INFO)
+
+
+def clean_notebook(notebook: Dict[str, Any]) -> bool:
+    """
+        Clean the given synapse notebook
+
+        :param notebook: The notebook json to clean
+        :return: `True` if the notebook had any of its properties cleaned, `False` otherwise
+    """
+    modified_attribute = False
+    for cell in notebook["properties"]["cells"]:
+        if "execution_count" in cell:
+            if cell["execution_count"] is not None:
+                cell["execution_count"] = None
+                modified_attribute = True
+    return modified_attribute
+
+
+def clean_notebooks(notebooks_to_clean: List[str]):
+    """
+        Clean all of the given notebooks
+
+        :param notebooks_to_clean: The list of notebook paths to clean
+    """
+    for notebook_path in notebooks_to_clean:
+        logging.info(f"Cleaning notebook '{notebook_path}'")
+        notebook = json.load(open(notebook_path))
+        cleaned_notebook = clean_notebook(notebook)
+        if cleaned_notebook:
+            logging.info("    Some properties of this notebook have been cleaned")
```

Comment:
Here can we show what was cleaned ? or how many cells where affected ?

Reply:
We could show the json diff, but we'd need to import a new package to do it which i want to avoid. Imo there's not much value in the json diff, since the diff can be easily shown in vscode/gitub. I could easily extend the code to output the number of modified cells, so i'll add this in